CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Add Kaoto view container with initial Help & Feedback view
 - Improve toggle source code (open/close camel file in a side textual editor) using one button
 - Add Integrations view into Kaoto view container
+- Remove legacy support for Kaoto v1 file patterns (*.kaoto.yaml,*.kaoto.yml)
```

Comment:
Should we prefix this with `Breaking`?

Reply:
so lets add ?

- `Breaking:` Remove legacy support for Kaoto v1 file patterns (*.kaoto.yaml,*.kaoto.yml)

Reply:
added prefix and moved note at a first position in a list